app/apps/server/bridges/users.js

```diff
@@ -14,4 +17,14 @@ export class AppUserBridge {
 
 		return this.orch.getConverters().get('users').convertByUsername(username);
 	}
+
+	async getActiveUserCount() {
+		return Users.find({
+			active: true,
+			$or: [
+				{ 'federation.peer': Federation.localIdentifier },
+				{ federation: { $exists: false } },
+			],
+		}).count();
```

Comment:
@sampaiodiego is this going to be safe to call at any sort of interval?

Reply:
nope.. I was hoping to have a better alternative for this by tomorrow.. 

@alansikora when can we expect to have the new flag? :grimacing:

Reply:
If we can get a better way to do this before the release, would surely be best 😬

Reply:
@sampaiodiego @alansikora any news on this?

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/15004

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/15004 has been merged! you can rely on `isRemote` field on `users` collection 😉